cpp/include/cudf/strings/convert/convert_durations.hpp

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/column/column.hpp>
+#include <cudf/strings/strings_column_view.hpp>
+
+namespace cudf {
+namespace strings {
+/**
+ * @addtogroup strings_convert
+ * @{
+ */
+
+/**
+ * @brief Returns a new duration column converting a strings column into
+ * durations using the provided format pattern.
+ *
+ * The format pattern can include the following specifiers:
+ * "%%,%n,%t,%D,%H,%I,%M,%S,%p,%R,%T,%r,%OH,%OI,%OM,%OS"
+ *
+ * | Specifier | Description | Range |
+ * | :-------: | ----------- | ---------------- |
+ * | %%%% | A literal % character | '%' |
+ * | %%n | A newline character | '\n' |
```

Comment:
Doxygen strips off the first one when formatting the web page.
Example: https://github.com/rapidsai/cudf/blob/9a89887dd34b479cc8e1939af0b506692d6a28dd/cpp/include/cudf/strings/convert/convert_datetime.hpp#L36
converts to:
https://docs.rapids.ai/api/libcudf/stable/group__strings__convert.html#gacaf50d3b6d14ece094b9fc16b3d35c76

Reply:
because `%%` was use in similar file. 
https://github.com/rapidsai/cudf/blob/9a89887dd34b479cc8e1939af0b506692d6a28dd/cpp/include/cudf/strings/convert/convert_datetime.hpp#L36

updated now.